src/test/java/oop/beverages/BeveragesTest.Java

```diff
@@ -0,0 +1,39 @@
+package oop.beverages;
+
+/**
+ * New class that defines all Lattes from Coffee
+ */
+public final class Latte extends Coffee{
```

Comment:
to be honest I'm surprised this .class file compiles - if it's 'BeveragesTest.java' the class name should match, it shouldn't be public final class Latte